src/Page/Guides.elm

```diff
@@ -0,0 +1,37 @@
+module Page.Guides exposing (view)
+
+import Dict
+import Html.Styled exposing (Html, a, div, h1, li, text, ul)
+import Html.Styled.Attributes exposing (class, href)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (translate)
+import Message exposing (Msg)
+import Page.Guide.Data exposing (Guide)
+import Route
+import Shared exposing (Model)
+
+
+view : Model -> Html Msg
+view model =
+    let
+        t =
+            translate model.language
+
+        guides =
```

Comment:
This should probably just be included in the main body of the function or in it's own. It's fine here for now though.

Generally better in Elm not to use the let unless it messes up the code readability and is unavoidable. It's bad for error messages and should also make sure we include type signatures even with these internal functions. Elm review will catch these anyway, later.